policies/lib/tree/30_generic_methods/file_from_local_source_recursion.cf

```diff
@@ -44,31 +44,35 @@ bundle agent file_from_local_source_recursion(source, path, recursion)
       # depth_search prevents from copying file/symlink
       # so we need to detect if we want to copy a folder or a file
       # If we check if it is a file, the class will always be true
-      # event if was in fact a directory. Then, we test if it is a dir.
-      "is_dir_copy"  expression => isdir("${path}");
+      # even if was in fact a directory. Then, we test if it is a dir.
+      "is_dir_copy"  expression => isdir("${source}");
```

Comment:
I think this should be suffixed by method_id

Reply:
I think that it is not needed, but we can add it, just in case